src/core/scene/a-scene.js

```diff
@@ -10,6 +10,7 @@ var utils = require('../../utils/');
 // Require after.
 var AEntity = require('../a-entity');
 var ANode = require('../a-node');
+var CustomEvent = require('custom-event');
```

Comment:
I don't think this is necessary. If the linter is complaining you can just add `CustomEvent` here: https://github.com/aframevr/aframe/pull/4199/files#diff-eb9bb1084863a2f65678ea67d6e7cf5eR1

Reply:
Got it!